src/scene/scenegraphnode.cpp

```diff
@@ -595,6 +652,91 @@ void SceneGraphNode::setDependencies(const std::vector<SceneGraphNode*>& depende
     }
 }
 
+void SceneGraphNode::computeScreenSpaceData(RenderData& newData) {
+    // Purposely slow the update rate of screen space position in order to reduce the
+    // effects of jittering in the position of information icon markers in web gui.
```

Comment:
Is this really the reason? I thought it would be for performance reasons?

Reply:
It is;  sending updates too often slows down the rendering, but also overloads CEF as can't handle that many updates iirc